Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/survey/SurveySettings.kt

```diff
@@ -1,7 +1,10 @@
 package de.rki.coronawarnapp.datadonation.survey
 
+import de.rki.coronawarnapp.datadonation.storage.DataDonationPreferences
 import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class SurveySettings @Inject constructor()
+class SurveySettings @Inject constructor(
```

Comment:
Do we need the extra level? Could we just use `SurveySetting` or delete it and just use the new preference class you created?

Reply:
see https://github.com/corona-warn-app/cwa-app-android/pull/2232#discussion_r566807152 for multi-use of this preferences class, if you are against is, i could skip this layer, but if it were up to me it would stay, because every layer improves finding a way through project structure

Reply:
obsolete as of https://github.com/corona-warn-app/cwa-app-android/pull/2232#discussion_r566847377

Reply:
I currently only see the `DataReset` class where we'd need access to both Analytics and Survey data structures, an abstraction layer just for that, seems a bit much. This wouldn't be a blocker for me though, maybe align with @kolyaopahle?

If the extra abstraction is desired, please reverse the order though.
* DataDonation (Parent/TopLevel)
   * Survey (Child)
   * Analytics (Child)